### PR TITLE
Added GnomeSort algorithm

### PR DESCRIPTION
**Pull request by TKF486**
added gnome sort algorithm, with added unit tests and updated DIRECTORY.md

### CODE REVIEW

File: Sorting/GnomeSort.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+			$a--;
+			if($a == 0){
+				$a = $b;
+				$b++;
+			}
+		}
+	}
+
+	return $array;
+}
```
Comment (darwinz) [inline Sorting/GnomeSort.php:34]: Will you please update the spacing in this file to use 4 spaces (or 1 tab) rather than 8 spaces (2 tabs)?

Comment (TKF486) [reply Sorting/GnomeSort.php]: I have fix the spacing issue, could you check if its ok now?

Comment (darwinz) [reply Sorting/GnomeSort.php]: That looks great!

**Review by darwinz — APPROVED** [review]
You'll need to rebase your branch so it's up to date with the master branch before we can merge, but looks great!

**Comment by TKF486** [comment]
Hi, i have just rebase my branch, could you help me check if is ok to merge now?

**Pull request merged**